src/transport/raw/PeerAddress.h

```diff
@@ -248,6 +257,7 @@ class PeerAddress
     uint16_t mPort               = CHIP_PORT; ///< Relevant for UDP data sending.
     Inet::InterfaceId mInterface = Inet::InterfaceId::Null();
     NodeId mRemoteId             = 0;
+    uint16_t mNFCShortId         = 0;
```

Comment:
I wonder whether it's worth putting mRemoteId and mNFCShortId in a union.  They're never used together, right?  Followup for that is best.

Reply:
Tracked in https://github.com/project-chip/connectedhomeip/issues/38615